packages/react-native/React/Fabric/Mounting/ComponentViews/ScrollView/RCTScrollViewComponentView.mm

```diff
@@ -468,7 +469,11 @@ - (void)updateState:(const State::Shared &)state oldState:(const State::Shared &
   }
 
   _contentSize = contentSize;
-  _containerView.frame = CGRect{RCTCGPointFromPoint(data.contentBoundingRect.origin), contentSize};
+  LayoutMetrics newContentContainerLayoutMetrics = LayoutMetrics{
+      .frame = {.origin = data.contentBoundingRect.origin, .size = data.getContentSize()}, .overflowInset = {.top = 1}};
```

Comment:
What's the `overflowInset` needed for?

Reply:
It needs to be set to anything but the default, so [this check](https://github.com/facebook/react-native/blob/900b625ee910cde61f9170e752020e974396aaae/packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm#L624) fails, `clipsToBounds` is set to false, and it passes the touches down to children which are out of bounds.